src/screens/WaitTimeScreen.tsx

```diff
@@ -1,42 +1,471 @@
+                  style={styles.tags}
+                  styles={StylesOverride.tagStyles}
+                >
+                  {type}
+                </Tag>
+              );
+            })}
+          </ScrollView>
+        </View>
+        {modalComponent}
+        <View style={styles.stepsContainer}>
+          <Steps
+            styles={StylesOverride.stepsStyles}
+            size="small"
+            direction="horizontal"
+          >
+            {steps1.map((_item: any, index: any) => (
+              <Step key={index} status={"wait"} />
+            ))}
+          </Steps>
+        </View>
+        <View style={styles.stepTextContainer}>
+          {steps1.map((item: any, index: any) => (
+            <View style={styles.stepTextView} key={index}>
+              <StyledText style={styles.stepTitleText}>{item.title}</StyledText>
+              <StyledText style={styles.stepDescriptionText}>
+                {item.description}
+              </StyledText>
+            </View>
+          ))}
+        </View>
+        <View style={styles.stepScrollDivider} />
+        <ScrollView
+          refreshControl={
+            <RefreshControl refreshing={refreshing} onRefresh={onRefresh} />
+          }
+        >
+          <List>
+            {poiData.map((poi: any, index: any) => (
+              <Item
+                key={index}
+                onPress={() => {
+                  navigation.navigate(LOCATION_DETAILS, {
+                    location: poi.location,
+                  });
+                }}
+                extra={
+                  <View>
+                    <Brief style={styles.poiWaitTimeText}>
+                      {poi.waitTime + " mins"}
+                    </Brief>
+                  </View>
+                }
+                arrow="horizontal"
+                thumb={
+                  <View style={styles.poiIcon}>
+                    {renderIcon(poi.type, theme.name)}
+                  </View>
+                }
+              >
+                <StyledText style={styles.poiDistanceText}>
+                  {poi.distance + " km"}
+                </StyledText>
+                <StyledText>{poi.location}</StyledText>
+                <StyledText style={styles.poiLastUpdatedText}>
+                  {renderLastUpdated(poi.lastUpdated)}
+                </StyledText>
+              </Item>
+            ))}
+          </List>
+        </ScrollView>
+      </View>
+    </TouchableWithoutFeedback>
   );
 };
 
 const styles = StyleSheet.create({
-  huge: {
-    fontSize: 20,
+  searchContainer: {
+    paddingTop: 20,
+    backgroundColor: "#FFFFFF",
   },
-  container: {
-    flex: 1,
-    backgroundColor: "#fff",
+  searchBar: {
+    borderRadius: 6,
+    backgroundColor: "#F5F5F5",
+  },
+  search: {
+    backgroundColor: "#F5F5F5",
+  },
+  sortContainer: {
+    paddingTop: 20,
+  },
+  tags: {
+    paddingRight: 6,
+  },
+  sortByTag: {
+    borderRadius: 24,
+    width: "20%",
```

Comment:
Do we need this width here? I'm assuming removing it would just allow it to fit the size of the content, the words? Would that be better?

Reply:
Good catch forgot to remove it